ReactTask/src/components/CreateAppointment/InputsAppointment.js

```diff
@@ -11,20 +12,43 @@ import {
   StylBoxColumnFlexMargin,
 } from './../';
 
-import { nameDoctorsArr, ocuppationDoctorArr } from './../../constants/index';
+import { getDoctorsBySpecializations } from '../../redux/patient/patientOperations';
 
-const InputsAppointment = ({ setFilledData }) => {
+const InputsAppointment = ({ setFilledData, setDoctorChoose }) => {
+  const [isOcuppationDoctorArr, setOcuppationDoctorArr] = useState([]);
+  const [isNameDoctorsArr, setNameDoctorsArr] = useState([]);
   const [isFocusInput, setFocusInput] = useState(false),
     [isFocusInputName, setFocusInputName] = useState(false),
     [isValueInputOccupation, setValueInputOccupation] = useState(''),
     [isValueInputName, setValueInputName] = useState('');
 
+  useEffect(() => {
+    getAllSpecializations().then((response) =>
+      setOcuppationDoctorArr(
+        response.data.map((item) => ({
+          value: item.id,
```

Comment:
Rodion said we need create hook to get rid of this logic in component

Reply:
Created new hooks